packages/DropdownMenu/src/components/Content/Content.js

```diff
@@ -0,0 +1,5 @@
+export default function Content() {
+  return null;
```

Comment:
Can you return <React.Fragment></React.Fragment> this helps React DocGen to recognize this as a ReactComponent and include it into the documentation, returning null will not be recognized as a react component.